tuna/miopen/miopen_lib.py

```diff
@@ -117,6 +118,9 @@ def parse_args(self):
     subcommands.add_subcommand('load_job',
                                get_load_job_parser(),
                                required=False)
+    subcommands.add_subcommand('export_db',
+                               get_export_db_parser(),
+                               required=False)
```

Comment:
in the run() function you need to call :
```
if self.args.subcommand is not None and self.args.subcommand == 'export_db':
      run_export_db([...])
      return None
```
which would be similar to run_import_configs and run_load_job!!! This will trigger the library to call your script code and not try and launch a worker.